src/components/NotificationBell.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { StyleProp, ViewStyle } from 'react-native'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { useNotifications } from 'src/home/NotificationBox'
+import NotificationBellIcon from 'src/icons/NotificationBellIcon'
+import { TopBarIconButton } from 'src/navigator/TopBarButton'
+import colors from 'src/styles/colors'
+
+interface Props {
+  style?: StyleProp<ViewStyle>
+  size?: number
+  testID?: string
+}
+
+export default function NotificationBell({ style, size, testID }: Props) {
+  const notifications = useNotifications()
+
+  const newNotifications = notifications.length > 0
+  const notificationMark = newNotifications ? colors.greenUI : undefined
+
+  const onPress = () => {
+    ValoraAnalytics.track(HomeEvents.notification_center_pressed, {
```

Comment:
This is ok? Or should I create a `NotificationEvents`?

Reply:
It's fine I think. But until we need this component in more than 1 place, can we keep it in `src/home`?
We usually move components to `src/components` when they are more generally useful.
But maybe that's the case for this one?

Reply:
agree - if a component is in the components folder then it should be generic (having analytics + navigation inside the onPress in this component makes it not generic / easy to reuse)